.gitignore

```diff
@@ -35,3 +35,11 @@ c/docker/netcode.io
 c/valgrind/netcode.io
 docs
 *.exe
+
+valgrind/*
+docker/*
+
+*.config
```

Comment:
This is under the ` # Directories we don't ever want to commit` section, should it be under ` # Misc` with the rest of the files?

Reply:
I didn't read, just threw something in.  Can fix it.